nodeadm/internal/aws/ecr/ecr.go

```diff
@@ -49,9 +49,8 @@ func GetEKSRegistry(region string) (ECRRegistry, error) {
 	}
 	if fipsInstalled && fipsEnabled {
 		fipsRegistry := getRegistry(account, "ecr-fips", region, servicesDomain)
-		if addresses, err := net.LookupHost(fipsRegistry); err != nil {
-			return "", err
-		} else if len(addresses) > 0 {
+		addresses, err := net.LookupHost(fipsRegistry)
+		if err == nil && len(addresses) > 0 {
```

Comment:
should we log whenever we skip this path? some other system log might catch this request, but i want it to be obvious when this path is not taken since i think `fipsEnabled` sets the expectation that the outcome will be fips